openwisp_radius/migrations/__init__.py

```diff
@@ -11,6 +11,12 @@ def get_swapped_model(apps, app_name, model_name):
     return apps.get_model(app, model)
 
 
+def delete_old_radius_token(apps, schema_editor):
+    # TODO:N Do we want to really delete all the old tokens?
+    # What consequenses should I think about?
```

Comment:
Just wanted to inform this change require deleting all old radius tokens, alternative would be allowing org value as `None`.

Reply:
currently they're disposable, they're created, consumed and destroyed right afterwards